client/layout/universal-navbar-header/nav-style.scss

```diff
@@ -28,6 +28,21 @@ $studio-blue-30: #0675c4;
 	}
 }
 
+.is-section-themes,
+.is-section-theme {
+	&.has-no-sidebar {
+		.masterbar {
+			height: 56px;
```

Comment:
It was unexpected to find that the `nav` element that is a direct child of the masterbar element has a height of 57px:

![Screenshot 2022-11-03 at 12 19 45](https://user-images.githubusercontent.com/8436925/199696565-1fdd3c0d-4d6f-46e9-9a8f-f352d7f469e5.png)

At the same time, masterbars throughout Calypso are generally 56px indeed. 

Thoughts?

Reply:
Makes sense, added the exception style rule.